doc_files/iter8_install.md

```diff
@@ -10,24 +10,24 @@ These instructions show you how to set up iter8 on Kubernetes with Istio.
 
 ## Install iter8 on Kubernetes
 
-iter8 has two components, _iter8_analytics_ and _iter8_controller_. To install them, follow the instructions below. For additional considerations when installing iter8 on Red Hat OpenShift, check out [these instructions](https://github.com/iter8-tools/docs/blob/v0.2.0/doc_files/platforms/redhat_openshift.md).
+iter8 has two components, _iter8_analytics_ and _iter8_controller_. To install them, follow the instructions below. For additional considerations when installing iter8 on Red Hat OpenShift, check out [these instructions](https://github.com/iter8-tools/docs/blob/v0.2/doc_files/platforms/redhat_openshift.md).
 
 ### Quick installation
 
 To install iter8 with the default settings, you can run the following install script:
 
 ```bash
-curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2.0/install/install.sh \
+curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2/install/install.sh \
 | /bin/bash -
 ```
 
 ### Customized installation via Helm charts
 
 In case you need to customize the installation of iter8, use the Helm charts listed below:
 
-* _iter8-analytics_: [ https://github.com/iter8-tools/iter8-analytics/releases/download/v0.2.0/iter8-analytics-helm-chart.tar](https://github.com/iter8-tools/iter8-analytics/releases/download/v0.2.0/iter8-analytics-helm-chart.tar)
+* _iter8-analytics_: [ https://github.com/iter8-tools/iter8-analytics/releases/download/v0.2/iter8-analytics-helm-chart.tar](https://github.com/iter8-tools/iter8-analytics/releases/download/v0.2/iter8-analytics-helm-chart.tar)
```

Comment:
This is not going to work. This is a link to a release artifact. We do not have a release v0.2.

Reply:
I agree. But, TBH, the other alternative doesn't make sense for someone using `latest` either since it won't correspond to the branch. I thought perhaps I should explicitly change it so that (a) it will be easy to find next time (just search for v0.2) and (b) problems using the link will remind one to use the source.